### DISCUSSION TITLE
`distinctKey` in the docs, but it does not exists

### DISCUSSION THREAD

**Discussion by kaosdev** [ReactiveX/rxjs#6142] [Report issues other than bug]
As per title, i found a **distinctKey** operator listed in the following page:
[ https://rxjs.dev/guide/operators]( https://rxjs.dev/guide/operators)

But it seems like this operator does not exists anymore.

**Comment by cartant** [comment]
From the 5.0.0-rc.2 CHANGELOG:

distinct: remove distinctKey, distinct signature change and perf improvements (#2049) (89612b2), closes #2009

So it should be removed from https://github.com/ReactiveX/rxjs/blob/0dff2dcf19d69738c7dd5594cd02db8742133744/docs_app/content/guide/operators.md

PRs welcome. 🙂

**Reply by kaosdev** [reply]
I can do that.

I also found an issue related to this. I think this could be closed after the PR.
https://github.com/ReactiveX/rxjs/issues/4980

EDIT:
Done the PR: https://github.com/ReactiveX/rxjs/pull/6146
